src/EnergyPlus/PollutionModule.cc

```diff
@@ -2188,3998 +419,214 @@ void SetupPollutionMeterReporting(EnergyPlusData &state)
-        }
-        Pollution.FuelOil2Comp.NucHiPollution = (FuelType.FuelOil2 * 1.0e-6) * FuelOil2Value;
-    }
-    if (Pollution.CoalCoef.FuelFactorUsed) {
-        Pollution.CoalComp.NucHiPollution = 0.0;
-        if (Pollution.CoalCoef.NucHiSched == 0) {
-            CoalValue = Pollution.CoalCoef.NucHi * 0.001;
-        } else {
-            CoalValue = Pollution.CoalCoef.NucHi * GetCurrentScheduleValue(state, Pollution.CoalCoef.NucHiSched) * 0.001;
-        }
-        Pollution.CoalComp.NucHiPollution = (FuelType.Coal * 1.0e-6) * CoalValue;
-    }
-    if (Pollution.GasolineCoef.FuelFactorUsed) {
-        Pollution.GasolineComp.NucHiPollution = 0.0;
-        if (Pollution.GasolineCoef.NucHiSched == 0) {
-            GasolineValue = Pollution.GasolineCoef.NucHi * 0.001;
-        } else {
-            GasolineValue = Pollution.GasolineCoef.NucHi * GetCurrentScheduleValue(state, Pollution.GasolineCoef.NucHiSched) * 0.001;
-        }
-        Pollution.GasolineComp.NucHiPollution = (FuelType.Gasoline * 1.0e-6) * GasolineValue;
-    }
-    if (Pollution.PropaneCoef.FuelFactorUsed) {
-        Pollution.PropaneComp.NucHiPollution = 0.0;
-        if (Pollution.PropaneCoef.NucHiSched == 0) {
-            PropaneValue = Pollution.PropaneCoef.NucHi * 0.001;
-        } else {
-            PropaneValue = Pollution.PropaneCoef.NucHi * GetCurrentScheduleValue(state, Pollution.PropaneCoef.NucHiSched) * 0.001;
-        }
-        Pollution.PropaneComp.NucHiPollution = (FuelType.Propane * 1.0e-6) * PropaneValue;
-    }
-    if (Pollution.DieselCoef.FuelFactorUsed) {
-        Pollution.DieselComp.NucHiPollution = 0.0;
-        if (Pollution.DieselCoef.NucHiSched == 0) {
-            DieselValue = Pollution.DieselCoef.NucHi * 0.001;
-        } else {
-            DieselValue = Pollution.DieselCoef.NucHi * GetCurrentScheduleValue(state, Pollution.DieselCoef.NucHiSched) * 0.001;
-        }
-        Pollution.DieselComp.NucHiPollution = (FuelType.Diesel * 1.0e-6) * DieselValue;
-    }
-    if (Pollution.OtherFuel1Coef.FuelFactorUsed) {
-        Pollution.OtherFuel1Comp.NucHiPollution = 0.0;
-        if (Pollution.OtherFuel1Coef.NucHiSched == 0) {
-            OtherFuel1Value = Pollution.OtherFuel1Coef.NucHi * 0.001;
-        } else {
-            OtherFuel1Value = Pollution.OtherFuel1Coef.NucHi * GetCurrentScheduleValue(state, Pollution.OtherFuel1Coef.NucHiSched) * 0.001;
-        }
-        Pollution.OtherFuel1Comp.NucHiPollution = (FuelType.OtherFuel1 * 1.0e-6) * OtherFuel1Value;
-    }
-    if (Pollution.OtherFuel2Coef.FuelFactorUsed) {
-        Pollution.OtherFuel2Comp.NucHiPollution = 0.0;
-        if (Pollution.OtherFuel2Coef.NucHiSched == 0) {
-            OtherFuel2Value = Pollution.OtherFuel2Coef.NucHi * 0.001;
-        } else {
-            OtherFuel2Value = Pollution.OtherFuel2Coef.NucHi * GetCurrentScheduleValue(state, Pollution.OtherFuel2Coef.NucHiSched) * 0.001;
+    // in progress (what is in progress?)
+
+    auto &pm = state.dataPollution;
```

Comment:
[src/EnergyPlus/PollutionModule.cc:538]:(style),[constVariable],Variable 'pm' can be declared as reference to const